lib/config.js

```diff
@@ -25,4 +27,5 @@ exports.GENERATING_STATIC_SITE = GENERATING_STATIC_SITE;
 exports.ENABLE_PUSHSTATE = ENABLE_PUSHSTATE;
 exports.IN_TEST_SUITE = (typeof describe == 'function');
 exports.DEV_SERVER_PORT = DEV_SERVER_PORT;
+exports.TWITTER_LINK = TWITTER_LINK;
```

Comment:
Cool. Just so you know, you're also welcome to just write `exports.TWITTER_LINK = "https://twitter.com/MozTeach"` if you want, it's up to you. I think I just started defining vars at the top so I could reuse them in the definition of other consts without having to prefix them with `exports.` or something.